docs/team/chinjunan.md

```diff
@@ -21,7 +21,7 @@ Given below are my contributions to the project.
     * Highlights: This feature required an addition to the data structure of the address book (an additional list to record reminders). This led to additional classes that involved managing the storage and reading from local memory and hard disk. Fortunately, not much have to be done as it was similar to existing codes. Further improvement to this feature involves running process threads to fire notifications while the app is running in the background.
     * Credits: Existing codes from AB3 which involved the models and storage functionality of the app.
 
-* **New Feature**: Added schedule reminder function.
+* **New Feature**: Added schedule notification function.
```

Comment:
btw can you add this in the UG - i think the UG only includes reminder feature

Reply:
Alright done!